Examples/Python/python/acts/examples/simulation.py

```
@@ -493,45 +483,38 @@ def addFatras(
+
+    s.addWhiteboardAlias("particles", outputParticles)
+
     return s
 
 
 def addSimWriters(
     s: acts.examples.Sequencer,
     simHits: Optional[str] = None,
-    particlesInitial="particles_initial",
-    particlesFinal="particles_final",
+    particlesSimulated="particles_simulated",
```

Ruff

Missing type annotation for function argument `particlesSimulated`

(ANN001)